hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t layer5/meshery:latest .
+docker tag layer5/meshery:latest layer5/meshery:$SOURCE_COMMIT
```

Comment:
Where does `$SOURCE_COMMIT` come from? I assume this is reference to the commit ID on GitHub sent via webhook on PR merge...